openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/Graph.vue

```diff
@@ -1366,7 +1407,17 @@ export default {
       }
     },
     subscriptionKey: function (item) {
-      return `DECOM__TLM__${item.targetName}__${item.packetName}__${item.itemName}__${item.valueType}`
+      // NONE maps to DECOM in the backend
+      let reduced = item.reduced === 'NONE' ? 'DECOM' : item.reduced
```

Comment:
This is confusing

Reply:
You don't like the ternary operator? :-)

Reply:
Actually this is a good catch. This was left over from a previous attempt. I'm mapping NONE to DECOM in the drop down so it's not needed. Will remove.